openshift-ci/enable_frrk8s_on_cno.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
```

Comment:
imo we should always include `set -euo pipefail` , I would like also a header what the script is doing

Reply:
re: header, isn't the name expressive enough?

Reply:
added